staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/conversion_test.go

```diff
@@ -76,12 +76,14 @@ func TestConvertSliceStringToDeletionPropagation(t *testing.T) {
 
 	for _, tc := range tcs {
 		var dp v1.DeletionPropagation
-		if err := v1.Convert_Slice_string_To_v1_DeletionPropagation(&tc.Input, &dp, nil); err != nil {
-			t.Errorf("Convert_Slice_string_To_v1_DeletionPropagation(%#v): %v", tc.Input, err)
+		dpPtr := &dp
```

Comment:
to accurately measure what the conversion method is doing now (which would have caught the nil-input short-circuit behavior change), this needs to start the output as nil

```
var dpPtr *v1.DeletionPropagation
```

and test like this:

```go
if !apiequality.Semantic.DeepEqual(dpPtr, &tc.Output) {
```

Reply:
done